src/components/Games/story-game/multi-choice-story-game.jsx

```
@@ -0,0 +1,273 @@
+
+  const restartGame = () => {
+    setCurrentQuestionIndex(0);
+    setSelectedAnswers({});
+    setShowResults(false);
+    setScore(0);
+    setShowExplanations(false);
+    if (onRestart) onRestart();
+  };
+
+  const isAnswered = (questionIndex) => {
+    return selectedAnswers.hasOwnProperty(questionIndex);
```

ESLint

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

(no-prototype-builtins)